api/envoy/extensions/filters/http/health_check/v3/health_check.proto

```diff
@@ -38,6 +38,11 @@ message HealthCheck {
   // If operating in non-pass-through mode, specifies a set of upstream cluster
   // names and the minimum percentage of servers in each of those clusters that
   // must be healthy or degraded in order for the filter to return a 200.
+  //
+  // .. note::
+  //
+  //    This value is interpreted as an integer, so 33.33% will be calculated
+  //    as if it were 33%.
```

Comment:
Can you clarify if this is truncation/floor function or rounding?

Reply:
Done, thanks.